TWLight/resources/models.py

```diff
@@ -439,6 +439,14 @@ def clean(self):
             raise ValidationError(
                 "Partners with automatically sent messages require user instructions to be entered"
             )
+        if (
```

Comment:
This looks good, but could be simplified like:
`if (self.authorization_method in [self.PROXY, self.BUNDLE])`